daycan-front/apps/admin/src/pages/login/components/index.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * 관리자 로그인 페이지에서 사용되는 컴포넌트들을 모아둔 곳입니다.
+ * @author admin-components
+ */
+export { ForgotCredentialsModal } from "./ForgotCredentialsModal";
```

Comment:
@author 에는 코드 작성해주신 보길님 이름 넣어주시면 좋을 것 같아요...!
```suggestion
/**
 * 관리자 로그인 페이지에서 사용되는 컴포넌트들을 모아둔 곳입니다.
 * @author 소보길
 */
export { ForgotCredentialsModal } from "./ForgotCredentialsModal";
```

Reply:
ai한테 맡겼다가 이 부분을 놓쳤습니다. 다음에는 제대로 주석처리 하겠습니다.!!